recipe/meta.yaml

```diff
@@ -1,16 +1,14 @@
 package:
   name: catboost
-  version: "0.8.1"
+  version: "0.9.1.1"
 
 build:
-  number: 0
+  number: 1
   #  Yandex supplies whl files on PyPI for:
-  # - Linux: Python 2.7, 3.4, 3.5 and 3.6
-  # - OS X: Python 2.7, 3.4, 3.5 and 3.6
-  # - Win-64: Python 2.7, 3.4, 3.5 and 3.6
+  # - Linux: Python 2.7, 3.4, 3.5, 3.6
+  # - OS X: Python 2.7, 3.4, 3.5, 3.6
+  # - Win-64: Python 2.7, 3.4, 3.5, 3.6
   skip: True  # [win32]
```

Comment:
FYI we also don't build on win32 any more.

Reply:
Thank you, Christopher. We'll remove this line next version update.